drop-in/src/main/java/com/adyen/checkout/dropin/ComponentParsingProvider.kt

```diff
@@ -399,7 +401,7 @@ internal fun getComponentFor(
         BcmcComponent.PROVIDER.isPaymentMethodSupported(paymentMethod) -> {
             val bcmcConfiguration: BcmcConfiguration =
                 getConfigurationForPaymentMethod(paymentMethod, dropInConfiguration)
-            BcmcComponentProvider(dropInParams).get(
+            BcmcComponentProvider(dropInParams, checkoutSession).get(
```

Comment:
Let's pass it everywhere not just to card and bcmc, to keep the provider class consistent

Reply:
If we passed this everywhere then I guess we would want to pass the whole checkoutSession object in case we needed it in the future WDYT?

Reply:
Yes indeed let's pass the same param everywhere (whether it's `CheckoutSession` or `SessionSetupConfiguration`) to keep providers consistent